CHANGELOG.md

```diff
@@ -21,6 +21,7 @@
 * Add `showSortIndicator` prop to `SearchAndSort` to display sort indicator next to the column names. STSMACOM-849.
 * Trim value to validate required field in the `ControlledVocab` component. STSMACOM-850.
 * Return a specific error message in `StripesConnectedSource`. STSMACOM-851.
+* DateRangeFilter - update all calls of `validateDateRange`. STSMACOM-853.
```

Comment:
Update them how? This comment amounts to "change function". We don't need paragraphs and footnotes, but enough context so we don't _have_ to read the code to grok the change would be helpful.